index.js

```diff
@@ -127,7 +134,12 @@ minimum value for how long a non-cached analyses will take
       try {
         result = await analysisPoller.do(requestResponse.uuid, this.accessToken, this.apiUrl, timeout, initialDelay, options.debug)
       } catch (e) {
-        if (e.status !== 401) {
+        /*
+          Normally requester passes back strings. However there is a spcial case for
+          HTTP 401 JWT access token has expired and we need to refresh it.
+        */
+        // console.log('\n\n\nXXX3\n\n');
+        if (e.hasOwnProperty('statusCode') && e.statusCode !== 401) {
```

Comment:
analysisPoller uses`omni-fetch` library which is quite different from `request`. It doesn't have `statusCode` but instead called `status`.

Reply:
Thanks for tracking this down. 

Re `omin-fetch` versus `request`: so many standards to choose from!

For handling specifically 401 one of these has to change, and I guess `lib/refresh.js` seems to be like the right candidate. 

So, yes, I would be grateful if you would change that. But just for handling 401 please. Or other things this should stay a string. Thanks.

Reply:
`omni-fetch` and `isomorphic-fetch` are a better way to go, because they follow https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API which is natively supported in browsers, and those libraries also work both in browsers and NodeJS.

Reply:
Yes, I'll add that as an issue. For now though since we want to get this out and have a lot of other things to do, this is more expedient.

Reply:
See https://github.com/ConsenSys/armlet/issues/90